linkerd/README.md

```diff
@@ -2,7 +2,10 @@
 
 ## Overview
 
-This check collects distributed system observability metrics from [Linkerd][1].
+Linkerd][1] is an ultra-light, ultra-simple, ultra-powerful open-source service mesh that provides developers with the tools they need to write secure, reliable, observable cloud-native applications. It is the only service mesh to reach CNCF graduated status. Linkerd uses mutual TLS to transparently secure all on-cluster TCP communication with no application changes, instantly adds latency-aware load balancing, request retries, timeouts, and blue-green deploys to keep your applications resilient, and provides instant platform health metrics by tracking success rates, latencies, and request volumes for every meshed workload — all with minimal configuration.
```

Comment:
```suggestion
[Linkerd][1] is an ultra-light, ultra-simple, ultra-powerful open-source service mesh that provides developers with the tools they need to write secure, reliable, observable cloud-native applications. It is the only service mesh to reach CNCF graduated status. Linkerd uses mutual TLS to transparently secure all on-cluster TCP communication with no application changes, instantly adds latency-aware load balancing, request retries, timeouts, and blue-green deploys to keep your applications resilient, and provides instant platform health metrics by tracking success rates, latencies, and request volumes for every meshed workload - all with minimal configuration.
```

Reply:
I'm wondering if you can split the third sentence into other sentences or a bulleted list?